_posts/2021-02-26-Starting-coding-on-Windows.md

```diff
@@ -0,0 +1,170 @@
+of setting up a development environment on Windows: many online bootcamps and
+starter programs assume or even require macOS or Ubuntu, with nary a mention of
+Windows at all. I thought I'd try to fill that gap :smile: For the purposes of
+this guide, I'm assuming Windows 10 Version 1903 (March 2019 Update) or higher.
+
+<!-- break-->
+
+## Working with the Command Line
+
+Many bootcamps and learn-to-code courses teach some command-line skills, or at
+least to feel comfortable in that environment. Historically, that's not
+something that developers on Windows have enjoyed much, limited as they were by
+cmd.exe and the arcane batch file programming language. In the last 10-15 years,
+however, PowerShell has been on the rise and last year reached version 7. It
+even runs cross-platform now, and with the Windows Subsytem for Linux, you can
+run Bash or zsh too! Most recently, the Windows console has had a ~~makeover~~
+rewrite, which brings it into the 21st centry with support for tabs, panes,
+themes (including image and transparent backgrounds), and more. Let's start with
+this.
+
+### Windows Terminal
+
+This is installed from the [Microsoft Store](https://aka.ms/terminal), which is
+the recommended method of installation to ensure it is kept up-to-date.
+Alternatively, you can [download it from
+GitHub](https://github.com/microsoft/terminal/releases/latest). You can choose
+to pin Windows Terminal to your Start screen/menu or task bar if you want to
+quick access to it.
+
+When you first run the terminal, it will open a Powershell tab; this will
+include a start-up message suggesting you "try the new cross-platform
+PowerShell". The version of PowerShell running in this tab is "Windows
+PowerShell", also known as the "Desktop Edition". It's two major versions out of
+date and has been superseded by PowerShell Core, which we'll install shortly.
+
+Windows Terminal is highly-customisable. Press <kbd>Ctrl</kbd>+<kbd>,</kbd> to
+open its settings file and edit it. The program will update its settings each
+time you hit Save! The documentation of the program and its settings is good, so
+be sure to [have a read of
+it](https://docs.microsoft.com/en-us/windows/terminal/) once you've finished
+this tutorial to get it working the way you want.
+
+### Package Manager
+
+In your freshly-minted PowerShell tab, run the following snippet to install
+Scoop, the missing package manager for Windows:
+
+```powershell
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
+iwr -useb get.scoop.sh | iex
+```
+
+The second line uses a bunch of aliases and other techniques in PowerShell to
+keep it short, but what it's doing is downloading a script from get.scoop.sh and
+executing it. It's quick and painless to install, and to use.
+
+A useful test of its installation is to install `sudo`, a handy PowerShell
+script that allows you to elevate your privileges when required. It's
+_compeletely different_ from `sudo` on Linux or macOS, but achieves the exact
```

Comment:
So it's not completely different if it does the same thing?

Reply:
How about this?
```suggestion
_compeletely different program_ from `sudo` on Linux or macOS, but achieves the exact
```